Sources/TrustSDK/ObjCWrapper/TrustSDKObjC.swift

```diff
@@ -0,0 +1,47 @@
+// Copyright © 2018 Trust.
+//
+// This file is part of TrustSDK. The full TrustSDK copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+#if XAMARIN
+import Foundation
+import TrustCore
+import BigInt
+
+@objc(TrustSDK)
+public extension TrustSDK {
+    @objc
+    public func signMessage(_ message: Data, address: String? = nil, completion: @escaping (Data) -> Void) {
+        guard WalletAppManager.hasWalletApp else {
+            return fallbackToInstall()
+        }
+        let addr = Address(string: address!)
+        let command = SignMessageCommand(message: message, address: addr, callbackScheme: callbackScheme, completion: completion)
+        execute(command: command)
```

Comment:
Could we call the other implementation instead of repeating the code?

Reply:
i could try again, but in my first attempts the binding failed if i did this